.github/workflows/build.yml

```diff
@@ -66,22 +66,25 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y --no-install-recommends \
-            ninja-build cmake g++ libgtest-dev libsdl2-dev zlib1g-dev libspdlog-dev
+            ninja-build cmake g++ libspdlog-dev
 
       - name: Install Windows dependencies
         if: ${{ matrix.os.runner == 'windows-latest' }}
         run: choco install ninja
-      
+
       - uses: ilammy/msvc-dev-cmd@v1
         if: ${{ matrix.os.runner == 'windows-latest' }}
         with:
            arch: win64
 
+      - uses: lukka/run-vcpkg@v11
+        with:
+          vcpkgGitCommitId: 2e7fd315d713e9eac52f6bdb24afbf1b4dbbe1e5
```

Comment:
awesome! will update. didn't know the difference there

Reply:
updated.

Reply:
Sweet! I would mark this conversation as resolved, but [I can’t because I don’t have write access to this repo](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations).